### PR TITLE
ui: update draggable list ux

### PR DESCRIPTION
**Pull request by Forfold**
**Description:**
- Updates flatlist drag and drop UX
- Updates Escalation Policy Steps List to use FlatList

**Which issue(s) this PR fixes:**
For significant amounts of work, it is best to start an issue first, preferably before the work is started.
For large pull requests, be sure to reference the associated GitHub issue(s).

Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.

**Out of Scope:**
- The caching update logic for steps flickers for a moment before "sticking"

**Screenshots:**

rotation

ep

### CODE REVIEW

**Review by allending313 — APPROVED** [review]
LGTM!

**Review by allending313 — APPROVED** [review]

File: web/src/cypress/e2e/escalationPolicySteps.cy.ts
[the remark this answers is not in the document]
Comment (allending313) [reply web/src/cypress/e2e/escalationPolicySteps.cy.ts:1]: resolved

**Comment by mastercactapus** [comment]
A couple quick questions:
- Is the flicker a regression introduced in this change or an existing bug?
- Are these still keyboard accessible?

**Comment by Forfold** [comment]
@mastercactapus The flicker is introduced for Escalation Policies as it is using the same caching logic but a new drag and drop library. It is probably for the best to address that as a follow-up ticket.

They are both still keyboard accessible (this is what the tests cover).

**Review by andrewbenington — APPROVED** [review]
works smoothly for me!

**Review by allending313 — APPROVED** [review]

**Pull request merged**